libs/designsystem/ng-package.json

```diff
@@ -11,5 +11,11 @@
       "@ionic/angular": "ionic"
     }
   },
-  "whitelistedNonPeerDependencies": ["@angular/cdk", "@ionic/angular", "highcharts", "moment"]
+  "whitelistedNonPeerDependencies": [
+    "@angular/cdk",
+    "@ionic/angular",
+    "highcharts",
+    "moment",
+    "@fontsource/roboto"
```

Comment:
I assume ng-packager / the publish complains without this? 🤔

Reply:
Yep - I have to whitelist the dependency on fontsource explicitly. It's a safeguard in order to not get unwanted dependencies 💂